Nikita_Chepurenko/21.09.21/ReversedArray.java

```diff
@@ -0,0 +1,24 @@
+package com.company;
+
+public class ReversedArray {
+    public int[] myReverse(int []array) {
+        int []reversedArrayList = new int[array.length];
```

Comment:
if (array.length == 0 || array == null ) {
       return .....;
     //throw new IllegaArgumentException();
   }
   int []reversedArrayList = new int[array.length];
        for (int index = array.length - 1; index >= 0; index--) {
            reversedArrayList[idx++] = array[index];
        }
        return  reversedArrayList;